internal/network/server.go

```diff
@@ -19,9 +19,12 @@ type ConnHandler func(Conn)
 type Server interface {
 	io.Closer
 
-	// Open opens the server on the given address. To choose the server a random
-	// free port for you, specify a port ":0".
+	// Open opens the server on the given address. To choose the
+	// server a random free port for you, specify a port ":0".
```

Comment:
Maybe this can be
"To allow the server to choose a random".... Seems a bit clearer.